### PR TITLE
[PT] Adding weeks 03 to 06

### PR DESCRIPTION
**Pull request by leonsolon**
[PT] First translation of weeks 03 to 06.

### CODE REVIEW

File: docs/pt/README-PT.md [hunk 1]
```diff
@@ -1,107 +1,107 @@
...
...ithub.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) &nbsp; [🇫🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/fr/README-FR.md) &nbsp; [🇮🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/fa/README-FA.md) &nbsp; [🇷🇺](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ru/README-RU.md) &nbsp; [🇻🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/vi/README-VI.md) &nbsp; [🇷🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/sr/README-SR.md) &nbsp; [🇵🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/pt/README-PT.md) &nbsp; [🇧🇩](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/bn/README-BN.md) &nbsp; [🇭🇺](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/hu/README-HU.md)
```
Comment (Atcold) [inline docs/pt/README-PT.md:10]: What happened here? Why the whole file has been rewritten?

File: docs/pt/index.md [hunk 2]
```diff
@@ -1,342 +1,342 @@
...
-Este curso trata das técnicas mais recentes em aprendizagem profunda e aprendizagem de representação, com foco em aprendizagem profunda supervisionada e não supervisionada, métodos de incorporação, aprendizagem métrica, redes convolucionais e recorrentes, com aplicações para visão computacional, compreensão de linguagem natural e reconhecimento de fala. Os pré-requisitos incluem: [DS-GA 1001 Introdução à Ciência de Dados](https://cds.nyu.edu/academics/ms-curriculum/) ou um curso de aprendizado de máquina de pós-graduação.
-
-## Aulas
-
-**Legenda**: 🖥 slides, 📓 Jupyter notebook, 🎥 vídeo do YouTube.
-
-<table>
-<!-- =============================== HEADER ================================ -->
-  <thead>
-    <tr>
```
Comment (Atcold) [inline docs/pt/index.md:29]: Same here. What's going on?

File: docs/pt/week01/01-1.md [hunk 3]
```diff
@@ -1,155 +1,155 @@
...
--->
-A história do aprendizado profundo remonta à um campo que mudou seu nome para cibernética agora. Começou na década de 1940 com McCulloch e Pitts. Eles tiveram essa ideia de que neurônios são unidades com limites, e com os estados ligado e desligado. Você poderia construir um circuito Booleano conectando os neurônios entre si e conduzir inferência lógica com neurônios. O cérebro é basicamente uma máquina de inferência lógica pois neurônios são binários. Eles computam uma soma ponderada das entradas e à comparam com o limiar. Ele liga se a soma é maior que o limiar, e desliga é menor, sendo uma visão simplificada de como redes neurais funcionam.
```
Comment (Atcold) [inline docs/pt/week01/01-1.md:43]: What's going on?

**Review by Atcold — CHANGES_REQUESTED** [review]
I cannot check what's going on if files are rewritten entirely.

image

Revert these changes, plz.

**Comment by leonsolon** [comment]
I'll send another one with corrected files!

**Comment by Atcold** [comment]
Oh, you could have simply sent a commit that was reverting those edits.

**Comment by leonsolon** [comment]
I was afraid to mess with your repo :D

805 is correct! Finally :P

**Comment by Atcold** [comment]
Okay, thanks.
No worries, I always check the PR content before merging.

**Pull request closed without merging**